sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample06_Transactions.md

```diff
@@ -44,6 +44,48 @@ using (var ts = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled))
 }
 ```
 
+### Transactions across entities
+In the previous version of the library, when attempting to perform transactions that spanned multiple entities, it was necessary to configure your senders so that they would send-via the other entity in the transaction. With updates to the service, the new library introduces the concept of transaction groups to handle cross entity transactions. When creating senders and receivers that should be part of a cross-entity transaction, you can specify the same `TransactionGroup` string in the options as shown below. 
```

Comment:
```suggestion
In the previous version of the library, when attempting to perform transactions that spanned multiple entities, it was necessary to configure your senders so that they would `send-via` the other entity in the transaction. With updates to the service, the new library introduces the concept of transaction groups to handle cross-entity transactions. When creating senders and receivers that should be part of a cross-entity transaction, you can specify the same `TransactionGroup` string in the options as shown below. 
```

Reply:
I'm thinking this discussion about send-via would be better suited for the migration guide. I will update to just give a brief conceptual overview of the feature in this sample.